[UX specs] inline validation errors (#2113)
3scale/porta: portafly/doc/ux_specs/global_components/alerts.md

# Alerts UX/UI specs

[Back to index](../index.md)

* [PatternFly Alerts design guidelines](https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/alerts-and-notifications)
* [PatternFly-react Alerts docs](https://www.patternfly.org/v4/documentation/react/components/alert)

### Input field validation errors / alerts

> * This type of alerts will be displayed as **inline alerts**

Reviewer: ```suggestion
* These types of alerts will be displayed as **inline alerts**
```

Reviewer: Depending on what you're trying to convey here, it could be: `* This type of alert will be displayed as an **inline alert**
`

Author: @dfennessy Thank you!

> * Validation errors – they block the form from being submitted successfully:

Reviewer: ```suggestion
* Validation errors – block forms with errors and/or omissions from being submitted:
```

Author: @dfennessy Thank you!
